src/In.ProjectEKA.HipService/appsettings.json

```diff
@@ -7,8 +7,9 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection":
-    "Host=db;Port=5432;Username=postgres;Password=welcome;Database=hipservice;",
-    "OTPGenerationConnection": "http://172.16.2.27:5000/otp/link",
-    "OTPVerificationConnection": "http://172.16.2.27:5000/otp/verify"
+    "Host=db;Port=5432;Username=postgres;Password=welcome;Database=hipservice;"
+  },
+  "OtpService": {
+    "BaseUrl":"http://localhost:5000"
```

Comment:
Did we solve the problem of HIP talking to OTP through docker-compose service in remote env?

Reply:
No, I tried searching for env variable unable to find any